package.json

```diff
@@ -76,5 +80,14 @@
   },
   "engines": {
     "node": ">=8"
+  },
+  "nyc": {
```

Comment:
Can you move this to a .nycrc file? I generally prefer to avoid dev tool config in package.json (though I realize it's reasonably common).